tests/istio.mk

```diff
@@ -59,12 +59,22 @@ EXTRA_E2E_ARGS += --ca_hub ${HUB}
 
 # A make target to generate all the YAML files
 generate_yaml:
-	./install/updateVersion.sh >/dev/null 2>&1
+	./install/updateVersion.sh -a ${HUB},${TAG} >/dev/null 2>&1
+
+# Generate the install files, using istioctl.
+# TODO: make sure they match, pass all tests.
+# TODO:
+generate_yaml_new:
+	./install/updateVersion.sh -a ${HUB},${TAG} >/dev/null 2>&1
+	(cd install/kubernetes/helm/istio; ${ISTIO_OUT}/istioctl gen-deploy -o yaml --values values.yaml)
 
 # Simple e2e test using fortio, approx 2 min
 e2e_simple: istioctl generate_yaml
 	go test -v -timeout 20m ./tests/e2e/tests/simple -args ${E2E_ARGS} ${EXTRA_E2E_ARGS}
 
+e2e_simple_run:
```

Comment:
ignore last comment regarding e2e test runs :)

Reply:
May be e2e_simple_run_test ?